### ISSUE TITLE
How to start the bot

### ISSUE THREAD

**Issue by alexsafayan** [Kenshii/Dribbble-Bot#1]
How do I start the bot?

**Comment by bolingerjosh** [comment]
You need to install Python on your machine (https://www.python.org/downloads/), then you can drag dribbble.py into the python application

**Comment by alexsafayan** [comment]
Thanks! But what do I do now?  See the image of the terminal:
<img width="609" alt="screen shot 2016-08-08 at 9 19 09 am" src="https://cloud.githubusercontent.com/assets/8205819/17484914/48c3d8f0-5d49-11e6-97f4-e416befa8018.png">

**Comment by jobolin** [comment]
Try this....

- delete the entire bot folder then redownload it from github
- open dribbble.py with XCode or TextEdit
- manually change the login and password lines by entering your dribbble credentials. Example:
```
	'login': 'myemail@mac.com',
	'password': 'mypassword'
```
- Save the file, then drag it into the Python Launcher app
- from here you will probably get warnings that you need to install certain packages. Look up the packages in google and install each of them. Drag dribbble.py after you install each package until it works

**Comment by jobolin** [comment]
Also, open Python Launcher and make sure it has the default options. Then when you drag the dribbble.py file, drop it right onto the Python Launcher icon on your dock

<img width="424" alt="pythonlauncher_default" src="https://cloud.githubusercontent.com/assets/3770284/17491622/01f63650-5d6e-11e6-9a6a-a1cab4062756.png">

**Comment by alexsafayan** [comment]
Hmm, it didn't ask me to download any packages, instead it gave me this:
<img width="607" alt="screen shot 2016-08-08 at 10 02 32 pm" src="https://cloud.githubusercontent.com/assets/8205819/17504195/e6bb20d0-5db3-11e6-9fd6-c06950d62f76.png">

**Comment by alexsafayan** [comment]
@jobolin Thank you so much for your help, but do you know what to do now?

**Comment by marekdlugos** [comment]
@alexsafayan then you want to make sure that you have installed `pip` and write to your terminal following:

`$ sudo pip install requests, re, time, json, os`

Usually when you get error messages try to Google them first and you will find a lot of useful answers.

**Comment by alexsafayan** [comment]
@marekdlugos Thanks, but when I enter my password to allow the install, I get 

sudo: pip: command not found

**Comment by marekdlugos** [comment]
@alexsafayan yes, so as I mentioned. Then you will copy your error message to Google which will give you [this site](http://stackoverflow.com/questions/9780717/bash-pip-command-not-found).

So solution for you is: `sudo easy_install pip`

**Comment by alexsafayan** [comment]
Oops, sorry!  I'm new to coding, I do that next time.  

So you want me to put `sudo easy_install pip` into the terminal, then `sudo pip install requests, re, time, json, os` ?

**Comment by marekdlugos** [comment]
Yes please. Don't worry. I was the same at the beginning and I'm still rookie. :)

So first
`sudo easy_install pip`
Then
`sudo pip install requests, re, time, json, os`

And lastly you should be able to run that script.

**Comment by alexsafayan** [comment]
Ok so I actually had to do `sudo -h pip install requests, re, time, json, os` , but even then it comes with this:

`usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
`

I looked it up and the closest thing I could find is [this,  ](https://github.com/github/git-lfs/issues/596)  but I don't think that applies(?)

**Comment by marekdlugos** [comment]
can you post the screenshot of terminal here?

**Comment by alexsafayan** [comment]
Yep!  Here's a screenshot:

<img width="651" alt="screen shot 2016-10-14 at 11 03 11 pm" src="https://cloud.githubusercontent.com/assets/8205819/19407531/76f1540c-9262-11e6-8676-ffba1e468c7f.png">

**Comment by marekdlugos** [comment]
I'm not sure now but do you actually need that `-h` there? I think that -h stands for `--help` which will return you just usage of pip command.

**Comment by alexsafayan** [comment]
Yes, maybe, but when I don't put the `-h` there, I get this:

<img width="656" alt="screen shot 2016-10-14 at 11 08 56 pm" src="https://cloud.githubusercontent.com/assets/8205819/19407566/4c3b663e-9263-11e6-99df-0852793eb34f.png">

**Comment by marekdlugos** [comment]
I'm not sure. You can try it install it one by one.

`sudo pip install requests`
`sudo pip install re`
...

If that wouldn't work I don't know.
